README.md

```diff
@@ -278,7 +278,7 @@ f(1);
 f("one");
 ```
 
-For more details, see [dtslint](https://github.com/Microsoft/dtslint#write-tests) readme.
+For more details, see [dtslint](https://github.com/Microsoft/DefinitelyTyped-tools/tree/master/packages/dtslint#write-tests) readme.
```

Comment:
BTW you could probably revert this if you wanted at this point, as the CI issue should have passed by now.

Reply:
It does make the link correct, while the status quo is incorrect (scroll up at the target).

Reply:
Sure, it's just not related to this PR.

Reply:
If it's a blocker to acceptance, I'll split it out; otherwise I'll leave it in to try to minimize unnecessary costs of contribution to open-source.

Reply:
Split out is better, as if we need to revert this PR, it'll also take this fix with it (which won't be desired). I'm happy to approve this change individually once it's sent in!

Reply:
Cherry-picked to #58952 which could be merged first, but I don't know what the bot's "Update support window diagram" commit is about.